Mathlib/RingTheory/DedekindDomain/FiniteAdeleRing.lean

```diff
@@ -266,6 +273,35 @@ theorem one : (1 : K_hat R K).IsFiniteAdele := by
   convert finite_empty
 #align dedekind_domain.prod_adic_completions.is_finite_adele.one DedekindDomain.ProdAdicCompletions.IsFiniteAdele.one
 
+open scoped DiscreteValuation
+
```

Comment:
Can you add a bit of doc, either here or at the top of the file (or at both!) with these new additions?

Reply:
I've added "We give this (finite adele) ring a K-algebra structure". Is this enough?